app/models/open_hour.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: open_hours
+#
+#  id         :bigint           not null, primary key
+#  close      :time
+#  day        :integer
+#  open       :time
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  seller_id  :bigint           not null
+#
+# Indexes
+#
+#  index_open_hours_on_seller_id  (seller_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (seller_id => sellers.id)
+#
+class OpenHour < ApplicationRecord
+  belongs_to :seller
+  validates_presence_of :day, :close, :open
+  # 1:Monday - 7:Sunday
+  validates_inclusion_of :day, in: 1..7
```

Comment:
Would it be possible to make this 0..6? This way we can X mod 7 vs (X mod 7) + 1. In general starting a zero is just going to be more natural for programmers